iree/compiler/Conversion/HLOToLinalg/HLOToLinalgOnTensors.cpp

```diff
@@ -42,15 +42,53 @@ namespace mlir {
 namespace iree_compiler {
 namespace {
 
+// Convert mhlo.is_finite to linalg.generic for float32 values ranked inputs.
```

Comment:
THis doesnt need to be here. Could you add this to TF MHLO -> Linalg conversion.

Reply:
This is going to be correct only for HW implementing IEEE 754 float arithmetic where f32 inf value = ` 0x7F800000` that was my concern regarding having this in TF.

Reply:
I think TF uses values like PI etc.  I would guess they also have some place they use for INF. Could we check with TF folks. If they dont, then we can put this in IREE

Reply:
SGTM! I will get in touch with TF folks.